File: Java/프로그래머스/0/181831. 특별한 이차원 배열 2/특별한 이차원 배열 2.java
class Solution {
    public int solution(int[][] arr) {
        int answer = 0;
        int count=0;
        for(int i=0; i<arr.length ; i++){
            for(int j=0; j<arr[i].length; j++){
                if(arr[i][j]==arr[j][i]) count++;
            }
        }
        if(count==(arr.length*arr[0].length)) answer=1;
        return answer;
    }
}

File: 프로그래머스/0/181831. 특별한 이차원 배열 2/특별한 이차원 배열 2.js
function solution(arr) {
    var answer = 0;
    let count=0;
    for(let i=0; i<arr.length ; i++){
        for(let j=0; j<arr[i].length; j++){
            if(arr[i][j]==arr[j][i]) count++;
        }
    }
    if(count==(arr.length*arr[0].length)) answer=1;
    return answer;
}